app/views/admin/_menu.html.haml

```diff
@@ -0,0 +1,38 @@
+%ul#slide-out.sidenav.sidenav-fixed
+  %li
+    .user-view
+      = image_tag 'U_can_act_logo_CMYK_GROEN.png', class: 'circle'
+    = link_to( admin_path, class: 'waves-effect') do
+      %span.black-text Admin interface 
+  %li
+    = link_to( admin_path, class: 'waves-effect') do
+      %i.material-icons> dashboard
+      Dashboard
+  %li
+    = link_to( admin_export_path, class: 'waves-effect') do
+      %i.material-icons> import_export
+      Exports
+  %li
+    = link_to( admin_preview_overview_path, class: 'waves-effect') do
+      %i.material-icons> question_answer
+      Preview questionnaires
+  %li
+    = link_to( admin_organization_overview_path, class: 'waves-effect') do
+      %i.material-icons> apps
+      Organization overview
+.fixed-action-btn.hide-on-large-only
+  %a.btn-floating.btn-large.accent-color.sidenav-trigger{"data-target" => "slide-out", :href => "#"}
+    %i.material-icons menu
+  %ul
+    %li
+      = link_to( admin_path, class: 'btn-floating') do
+        %i.material-icons> dashboard
+    %li
+      = link_to( admin_export_path, class: 'btn-floating') do
+        %i.material-icons> import_export
+    %li
+      = link_to( admin_preview_overview_path, class: 'btn-floating') do
+        %i.material-icons> question_answer
+    %li
+      = link_to( admin_organization_overview_path, class: 'btn-floating') do
+        %i.material-icons> apps
```

Comment:
Misschien ook linkjes includen naar de dummy apps? Ik weet dat je daar geen admin login voor nodig hebt enzo, maar ik denk voor Nick en Mandy (of misschien juist ook voor andere partijen die de app gaan gebruiken) is het misschien wel fijn om die URLs niet steeds te hoeven intypen oid. Of puur omdat het nu hidden functionality is die nergens geexposed wordt (je moet het maar net weten).

Misschien anders een linkje "Dummy applications" dat naar een pagina gaat met daarop de links naar de mentor dummy app en de student dummy app zodat mensen die makkelijk kunnen zien/copy/pasten oid?